### ISSUE TITLE
BY 07/24: Turn on feature flag for pilot 

### ISSUE THREAD

**Issue by abbyraskinUSDS** [department-of-veterans-affairs/caseflow#5867]
Enable AMA features for two Board users in time for user testing, now scheduled for Thursday 07/26 at 11am ET.
- Ivy Wilson: bvaimbutler
- Rachel Sauter: bvarsauter

**Comment by abbyraskinUSDS** [comment]
We should enable the AMA features for two users:
- Rachel Sauter
- Ivy Wilson

**Comment by shanear** [comment]
Because of the BGS errors, the pilot will have to be postponed till Monday July 23.

**Comment by leikkisa** [comment]
@abbyraskinUSDS Can I get their usernames?

**Comment by abbyraskinUSDS** [comment]
I don't have them, but I'll ask and update the ticket. Ivy is out today.

**Comment by abbyraskinUSDS** [comment]
FYI the session was rescheduled for next Thursday morning (obviously still pending BGS being ready by then).

**Comment by abbyraskinUSDS** [comment]
Added Rachel's, still waiting on Ivy

**Comment by abbyraskinUSDS** [comment]
@kahlouie @leikkisa i added the user IDs

**Comment by leikkisa** [comment]
@abbyraskinUSDS Hey Abby! Is it reasonable to ask if one of the users can check it out tomorrow to see if it's working?

**Comment by abbyraskinUSDS** [comment]
I confirmed that Ivy and Rachel have access now. And I do too. :)
